src/backend/interfaces/REST/go.server/api_server.go

```diff
@@ -77,37 +77,35 @@ func (server *REST_API) start() error {
 	// Creates a gin router with default middleware:
 	// logger and recovery (crash-free) middleware
 	router := gin.Default()
-        // adds our middlewares
-        router.Use(SwaggerInboundValidation())
-	router.Use(BindBackend())
+	// adds our middlewares
+	//router.Use(SwaggerInboundValidation())
 
-        // adds our routes and handlers
-        api := router.Group("/api/v2")
-        AddHandlers(api)
+	// adds our routes and handlers
+	api := router.Group("/api/v2")
+	AddHandlers(api)
 
-        // listens
+	// listens
 	addr := server.config.Host + ":" + server.config.Port
 	err := router.Run(addr)
-        if err != nil {
-                log.WithError(err).Info("unable to start gin server")
-        }
-	return err
-}
-
-func BindBackend() gin.HandlerFunc {
-	return func(c *gin.Context) {
-		c.Set("APIStore", server.store)
-		c.Next()
+	if err != nil {
+		log.WithError(err).Info("unable to start gin server")
```

Comment:
don't log info level if it's an error, warn at least

It's a generic remark, you sometimes log an error using info level